pkg/cluster/dao/dao.go

```diff
@@ -25,6 +25,7 @@ type DAO interface {
 	GetByID(ctx context.Context, id uint, includeSoftDelete bool) (*models.Cluster, error)
 	GetByName(ctx context.Context, clusterName string) (*models.Cluster, error)
 	UpdateByID(ctx context.Context, id uint, cluster *models.Cluster) (*models.Cluster, error)
+	UpdateTemplateByID(ctx context.Context, id uint, template, release string) (*models.Cluster, error)
```

Comment:
just using UpdateByID will be enough

Reply:
UpdateByID does not update template, now. Unnecessary problems may arise if restrictions are relaxed ？